reference/api/settings.mdx

```diff
@@ -1910,3 +1911,189 @@ Reset an index's typo tolerance settings to their [default value](#typo-toleranc
 ```
 
 You can use the returned `taskUid` to get more details on [the status of the task](/reference/api/tasks#get-one-task).
+
+## Embedders (experimental)
+
+Embedders translate documents and queries into vector embeddings. You must configure at least one embedder to use AI-powered search.
+
+### Embedders object
+
+The embedders object may contain an arbitrary number of embedder objects:
+
+```json
+{
+  "default": {
+    "source": "huggingFace",
+    "model": "BAAI/bge-base-en-v1.5",
+    "documentTemplate": "A movie titled '{{doc.title}}' whose description starts with {{doc.overview|truncatewords: 20}}"
+  },
+  "openai": {
+    "source": "openAi",
+    "apiKey": "OPENAI_API_KEY",
+    "model": "text-embedding-3-small",
+    "documentTemplate": "A movie titled {{doc.title}} whose description starts with {{doc.overview|truncatewords: 20}}",
+  }
+}
+```
+
+These embedder objects may contain the following fields:
+
+| Name                   | Type             | Default Value                                                            | Description                                                                                                                                                    |
+|--:---------------------|--:---------------|--:-----------------------------------------------------------------------|--:-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`source`**           | String           | Empty                                                                    | The third-party tool that will generate embeddings from documents. Must be `openAi`, `huggingFace`, `ollama`, `rest`, or `userProvided`                        |
+| **`url`**              | String           | `http://localhost:11434/api/embeddings`                                  | URL Meilisearch contacts when querying the embedder                                                                                                        |
```

Comment:
I understand the detailed meaning of the REST embedder parameters (url, inputField, inputType, query, pathToEmbeddings and embeddingObject) will be specified in the later guide on REST embedder.

Reply:
I'm not sure I understand your statement. Setting aside any other piece of content, is the definition here correct? Or is it correct for ollama/openAI but incorrect for REST embeddders?

Reply:
The definitions are not incorrect, but they lack context.